packages/react-components/src/components/SendBox.tsx

```diff
@@ -256,11 +288,24 @@ export const SendBox = (props: SendBoxProps): JSX.Element => {
     [mergedSendIconStyle, onRenderIcon]
   );
 
+  /* @conditional-compile-remove(file-sharing) */
+  const latestFileUploadErrorMessage = useMemo(() => {
+    return props.activeFileUploads?.filter((fileUpload) => fileUpload.errorMessage).pop()?.errorMessage;
+  }, [props.activeFileUploads]);
+
   return (
     <Stack className={mergeStyles(sendBoxWrapperStyles)}>
       {
         /* @conditional-compile-remove(file-sharing) */
-        renderFileUploadErrorMessage
+        <SendBoxErrors
+          {...{
```

Comment:
Why do we use the spread operator instead of put all properties directly into the component?

Reply:
Updated the code to make it more sensible. :)